src/components/bcros/filing/common/HeaderActions.vue

```diff
@@ -131,6 +131,7 @@ const tempBusinessIdentifier = computed(() => bootstrapFiling.value.filing.busin
 const filingId = computed(() => filing.value.filingId)
 const isTypeStaff = computed(() => isStaffFiling(filing.value))
 const isFutureEffectiveFiling = computed(() => isFutureEffective(filing.value))
+const isWithdrawalPending = computed(() => filing.value.withdrawalPending)
```

Comment:
Is `withdrawalPending` always present in the filing? If not, what's the value of isWithdrawalPending and what happens on line 340?

Reply:
Looks like it's always present. Never mind. Resolving.